invenio_app_rdm/config.py

```diff
@@ -494,6 +498,28 @@ def files_rest_permission_factory(obj, action):
 # is solved
 CSRF_HEADER = "X-CSRFToken"
 
+# Invenio-Vocabularies
+# =============
+
+VOCABULARIES_DATASTREAM_READERS = {
```

Comment:
@slint should I do this, or something like:

```python
if VOCABULARIES_DATASTREAM_READERS:
    VOCABULARIES_DATASTREAM_READERS["orcid-http"] = OrcidHTTPReader
else:
    VOCABULARIES_DATASTREAM_READERS= {
        "orcid-http": OrcidHTTPReader,
    }
```

in order to avoid missing "default" ones from `invenio-vocabularies... There was a similar method I recall for `invenio-oauthclient` config, but I did not manage to find it...

Reply:
Sometimes I think we just make a "deepcopy" and update the `dict`... that way we don't miss any default config that comes out of the box. So something like:

```python
from invenio_vocabularies.config import VOCABULARIES_DATASTREAM_READERS

VOCABULARIES_DATASTREAM_READERS = {
    **VOCABULARIES_DATASTREAM_READERS,
    "orcid-http": OrcidHTTPReader,
    ...
}
```

Reply:
Done :)